### PR TITLE
Update api-server-vnet-integration.md

### PR DESCRIPTION
**Pull request by rahulrai-in**
Added NSG rules required to enable communication between components.

### CODE REVIEW

**Comment by rahulrai-in** [comment]
Could you please review this change @chasewilson @feiskyer

**Comment by rahulrai-in** [comment]
Could you please review this change @chasewilson @feiskyer

File: articles/aks/api-server-vnet-integration.md [hunk 1]
```diff
@@ -262,6 +262,15 @@ AKS clusters configured with API Server VNet Integration can have public network
...
     ```
 
+## NSG security rules
+
+Ensure that the existing NSG security rules permit the following types of communication:
+
+| Destination        | Source                                                             | Protocol | Port    | Use  |
+|----------------------------------------------------------------------------------|----------|---------|------|
+| APIServer Subnet CIDR   | Cluster Subnet | TCP           | 443 and 4443      | Required to enable communication between Nodes and the API server.|
+| APIServer Subnet CIDR   | Load Balancer | UDP           | 9988      | Required to enable communication between Azure Load Balancer and the API server. You can also enable all communicatios between the Load Balancer and the API Server Subnet CIDR. |
```
Comment (feiskyer) [inline articles/aks/api-server-vnet-integration.md:272]:
```suggestion
| APIServer Subnet CIDR   | AzureLoad Balancer |  TCP           | 9988      | Required to enable communication between Azure Load Balancer and the API server. You can also enable all communicatios between the Azure Load Balancer and the API Server Subnet CIDR. |
```

File: articles/aks/api-server-vnet-integration.md [hunk 2]
```diff
@@ -262,6 +262,15 @@ AKS clusters configured with API Server VNet Integration can have public network
     az aks get-credentials --resource-group <resource-group> --name <cluster-name>
     ```
 
+## NSG security rules
+
+Ensure that the existing NSG security rules permit the following types of communication:
```
Comment (feiskyer) [inline articles/aks/api-server-vnet-integration.md:267]:
```suggestion
All traffic within the VNet is allowed by default. But if you have added NSG rules to restrict traffic between different subnets, ensure that the NSG security rules permit the following types of communication:
```

**Review by feiskyer — APPROVED** [review]

**Comment by Court72** [comment]
@asudbring

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by asudbring** [comment]
#sign-off

**Comment by Jak-MS** [comment]
@rahulrai-in @asudbring - the table was broken so I changed the markdown syntax to fix the issue, but please check that this is what you intended.:

| Destination | Source | Protocol | Port | Use |
|--- |--- |--- |--- |--- |
| APIServer Subnet CIDR   | Cluster Subnet | TCP           | 443 and 4443      | Required to enable communication between Nodes and the API server.|
| APIServer Subnet CIDR   | Azure Load Balancer |  TCP           | 9988      | Required to enable communication between Azure Load Balancer and the API server. You can also enable all communicatios between the Azure Load Balancer and the API Server Subnet CIDR. |

**Comment by Jak-MS** [comment]
If it was NOT, we can fix in a new PR. thanks.

**Pull request merged**